### ISSUE TITLE
Pointer delay; Explorer crash; reversion to small taskbar icons

### ISSUE THREAD

**Issue by CottonEaster** [ChanpleCai/SmartTaskbar#22]
Upon first run (of SmartTaskBar 1.1.8 - not of previous versions) the following happened: my mouse pointer became very slow; then explorer.exe crashed and restarted.

Also: since I installed the new version of SmartTaskbar, the size of my taskbar icons keeps reverting from normal to small, even though I do not have the 'small icons' option ticked in the options of SmartTaskbar. (Or at least I believe that I do not. Since the unticked state is represented with no icon at all, it is somewhat hard to tell.)

Version 1.1.8 running on Windows 8.1 Pro x64.

**Comment by ChanpleCai** [comment]
My God, I may have written a bug here.
Please try to download and install it again once to see the effect.
If it still doesn't work, use 1.1.7 first. I will check it out.

**Comment by ChanpleCai** [comment]
Can you specify whether it is in "Adaptive " or "Auto" mode?

**Comment by ChanpleCai** [comment]
I am currently using a Windows 8.1 x64 computer, so far I can't reproduce the bug you said. Could you give me more Information, such as when these exceptions are triggered?

**Comment by ChanpleCai** [comment]
About the first run crash. The first installation package I uploaded yesterday did have this problem. If you download and install again, it should not crash anymore.

As for the second question, if the currently running mode is Adaptive Mode, then it is normal.

**Comment by ChanpleCai** [comment]
The problem of the taskbar icon disappearing when changing the button size. This is the taskbar itself refreshing, the speed of the refresh depends on the hardware of the computer.I tried to solve this problem, but no progress = =

**Comment by CottonEaster** [comment]
Hello! The program showed both 'adaptive' and 'auto' as unchecked when I first ran it. I've set it to 'auto' a few hours ago and have had no problems since then (though I have not yet rebooted).

> As for the second question, if the currently running mode is Adaptive Mode, then it is normal.

Aha. Perhaps the program should by default start in 'auto' mode then?

> About the first run crash. The first installation package I uploaded yesterday did have this problem. If you download and install again, it should not crash anymore.

I've just tried the new installer. It seemed to work perfectly - but I'll see whether it continues to function. If it does, I'll close this issue. Thank you for your work.
